packages/frontend/src/components/common/ResourceIcon/index.tsx

```diff
@@ -224,9 +224,12 @@ export const ResourceIndicator: FC<
                     opened={opened || isHovering}
                 >
                     <MantineIcon
-                        {...iconProps}
+                        icon={iconProps.icon}
                         onMouseEnter={handleMouseEnter}
                         onMouseLeave={handleMouseLeave}
+                        style={{
+                            color: iconProps.color, // NOTE: If react-tabler icon is filled, then we have to override the color this way
```

Comment:
also works for non-filled 
<img width="497" alt="image" src="https://github.com/lightdash/lightdash/assets/7611706/6872b67e-ebca-4c09-9a83-9a52aa10caeb">